pkgs/development/interpreters/rakudo/moarvm.nix

```diff
@@ -1,39 +1,59 @@
 {
-  lib,
-  stdenv,
   fetchFromGitHub,
+  fetchpatch2,
+  lib,
   perl,
+  pkg-config,
+  stdenv,
+  versionCheckHook,
+  zstd,
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "moarvm";
-  version = "2025.06";
+  version = "2025.10";
 
   # nixpkgs-update: no auto update
   src = fetchFromGitHub {
-    owner = "moarvm";
-    repo = "moarvm";
-    rev = version;
-    hash = "sha256-QtJ8cLAbsFJ26wkfQCbIMVU1ArWlAXjsQ/RJbQ0wRNo=";
+    owner = "MoarVM";
+    repo = "MoarVM";
+    tag = finalAttrs.version;
     fetchSubmodules = true;
+    hash = "sha256-ZTQsjEHBAhmUdEJAKz/mt+DHcSwlY41jIMtJ2vFNNqY=";
   };
 
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ zstd ];
+
+  patches = [
+    (fetchpatch2 {
+      name = "fix-format-specifiers.patch";
+      url = "https://github.com/Prince213/MoarVM/commit/b937272f92c58278cca4c6188d91b0ee7a0027da.patch?full_index=1";
```

Comment:
Hi there @Prince213!

Can you please describe the nature of this patch retrieved from your moarvm fork, and why it is neccecary? Also, should it perhaps be upstreamed?

Reply:
Oh right, it's already merged as https://github.com/MoarVM/MoarVM/pull/1964.  I'll add a note.